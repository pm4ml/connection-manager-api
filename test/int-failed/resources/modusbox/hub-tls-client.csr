-----BEGIN CERTIFICATE REQUEST-----
MIIFfzCCA2cCAQAwgbIxCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJXQTEQMA4GA1UE
BxMHU2VhdHRsZTERMA8GA1UEChMITW9kdXNib3gxFDASBgNVBAsTC0VuZ2luZWVy
aW5nMR0wGwYDVQQDExRodWIuZGV2Lm1vZHVzYm94LmNvbTE8MDoGCSqGSIb3DQEJ
AQwtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWluQGh1Yi5kZXYubW9kdXNib3guY29t
MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA9aGgGgJkvv2JfWzvaei4
5+Vf7VicajqvEjqO/KjWQwZLUXRAq3shIm2onN67ftDWj5uo8k2CsR8l/bNPeML8
THyfq5OHrV1i/BX8V4/Dy82caOUnUoq0Rvl7daNSz7McrA+hvMjWD7r9FN8qO4o7
nGxLmZ6AMlThVwTcr8UGR9Z5tKC4RjhHljH5NTSWchmLZj+kQohnK1Gz9zT3ZRhA
ynEkv7jH2oi4YCJNMu/yKgCaUSH11JU+eHJTXePNqKslMNyAPR71aAkGyqJaOPHo
i833m9EkDTSPg+dJTzzW/y8/T7o+kvucFLoF+bpQ1LA8EgY4Z5jAxB9Hy7Nns+z+
Sw74S4+ad5KrXcqwdpHe+uhhRRBBC1bF4UHLQ5+kpvCY8is+KNUCcki8NFb0rh0L
BNK3vbU/a5iUlKCDmebtmB20TEfrLqGdS2b+CvNPDzHDTBt6m61FiA8M1WoWvzgE
8cjYyEG2/lZIQHJ6nzAJ0kFcal8mu7OibkQMEnxHQmsxMi4/NQ4i+SVSWqVb+n3o
YMSLvfOxe8kXYpnOulbbG87ocFL6Y/6ceRYHo9vLVxSVmuS6UOTbMF2JAFWRbkqI
RAOkEeipxvkHy6GLz9UhVMqxK4OmxQ1bLI/Gpk3ULPhBx1lXtyRFeBfk1YjGZlFp
sT9ar3pGdPcRCa0feFkUzOMCAwEAAaCBhjCBgwYJKoZIhvcNAQkOMXYwdDByBgNV
HREEazBpghVodWIxLmRldi5tb2R1c2JveC5jb22CFWh1YjIuZGV2Lm1vZHVzYm94
LmNvbYEtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWluQGh1Yi5kZXYubW9kdXNib3gu
Y29thwSjCgUYhwSjCgUVMA0GCSqGSIb3DQEBDQUAA4ICAQDzGBiYTjwgXnu1+y7y
8HRKLny0EYMpVOrAf/sSrVKsh2ExS3o10VzV+KyewPsSDfew/S7VDxWqZcJlZrEJ
XW/ZgZGkDjsVTgCSMQ/IVPDXKyQIEzCWB8Ne5UnC9oIjUlB5l3Svld4Q0DtS6hE6
3azz8TIYSivwIE8OG/TXQxqtjjJMC2sCcHA9KZsZZwes1vezXR3LIy4R8GBLYpVV
DGiQnNsXnIqcx2nH3kP/l00PK9Kk/jOJxKGbQbs79ntaAJ9AXuQHYxg13q4ppGG+
SCWkEx3LDQTPi+WWhlcETnpvkT1jAH1SWV3ld8c7TcXPpcj0DYTaBYovL6baeCCD
PDObXFU4A0QFivwgvrKicc85JWtg4Y10UCcPRfAxpnT8afV9KY+sX6oZo1cKcCWL
+SDc3Ikyd3Vf3nPvDy/ZqyGx/h5UpFKwgG9usYEzz57LVBxEnaOfRLN2E6sNvuse
rr1yejFQXRqxBPj0PniiNyRy+dsFeCIflAa/cmzYch/lWgpM8pEj4XsYSu7lRMqx
ihmpmqpClZYx0Mu4CTaHW85S0aV6EhMOZgJmRnm+R+QNB4Q44ssnfyeb9TipKCE0
kMCix1EqPoucgIekQzedwe32wDR0oGYMOdA1w44maDC9QaIKLjhu2f+8m+CqeD4Q
vr97InCotkF5yL/eQtiSkUhUjg==
-----END CERTIFICATE REQUEST-----
