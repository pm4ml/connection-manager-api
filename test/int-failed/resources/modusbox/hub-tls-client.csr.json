"-----BEGIN CERTIFICATE REQUEST-----\nMIIFfzCCA2cCAQAwgbIxCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJXQTEQMA4GA1UE\nBxMHU2VhdHRsZTERMA8GA1UEChMITW9kdXNib3gxFDASBgNVBAsTC0VuZ2luZWVy\naW5nMR0wGwYDVQQDExRodWIuZGV2Lm1vZHVzYm94LmNvbTE8MDoGCSqGSIb3DQEJ\nAQwtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWluQGh1Yi5kZXYubW9kdXNib3guY29t\nMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA9aGgGgJkvv2JfWzvaei4\n5+Vf7VicajqvEjqO/KjWQwZLUXRAq3shIm2onN67ftDWj5uo8k2CsR8l/bNPeML8\nTHyfq5OHrV1i/BX8V4/Dy82caOUnUoq0Rvl7daNSz7McrA+hvMjWD7r9FN8qO4o7\nnGxLmZ6AMlThVwTcr8UGR9Z5tKC4RjhHljH5NTSWchmLZj+kQohnK1Gz9zT3ZRhA\nynEkv7jH2oi4YCJNMu/yKgCaUSH11JU+eHJTXePNqKslMNyAPR71aAkGyqJaOPHo\ni833m9EkDTSPg+dJTzzW/y8/T7o+kvucFLoF+bpQ1LA8EgY4Z5jAxB9Hy7Nns+z+\nSw74S4+ad5KrXcqwdpHe+uhhRRBBC1bF4UHLQ5+kpvCY8is+KNUCcki8NFb0rh0L\nBNK3vbU/a5iUlKCDmebtmB20TEfrLqGdS2b+CvNPDzHDTBt6m61FiA8M1WoWvzgE\n8cjYyEG2/lZIQHJ6nzAJ0kFcal8mu7OibkQMEnxHQmsxMi4/NQ4i+SVSWqVb+n3o\nYMSLvfOxe8kXYpnOulbbG87ocFL6Y/6ceRYHo9vLVxSVmuS6UOTbMF2JAFWRbkqI\nRAOkEeipxvkHy6GLz9UhVMqxK4OmxQ1bLI/Gpk3ULPhBx1lXtyRFeBfk1YjGZlFp\nsT9ar3pGdPcRCa0feFkUzOMCAwEAAaCBhjCBgwYJKoZIhvcNAQkOMXYwdDByBgNV\nHREEazBpghVodWIxLmRldi5tb2R1c2JveC5jb22CFWh1YjIuZGV2Lm1vZHVzYm94\nLmNvbYEtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWluQGh1Yi5kZXYubW9kdXNib3gu\nY29thwSjCgUYhwSjCgUVMA0GCSqGSIb3DQEBDQUAA4ICAQDzGBiYTjwgXnu1+y7y\n8HRKLny0EYMpVOrAf/sSrVKsh2ExS3o10VzV+KyewPsSDfew/S7VDxWqZcJlZrEJ\nXW/ZgZGkDjsVTgCSMQ/IVPDXKyQIEzCWB8Ne5UnC9oIjUlB5l3Svld4Q0DtS6hE6\n3azz8TIYSivwIE8OG/TXQxqtjjJMC2sCcHA9KZsZZwes1vezXR3LIy4R8GBLYpVV\nDGiQnNsXnIqcx2nH3kP/l00PK9Kk/jOJxKGbQbs79ntaAJ9AXuQHYxg13q4ppGG+\nSCWkEx3LDQTPi+WWhlcETnpvkT1jAH1SWV3ld8c7TcXPpcj0DYTaBYovL6baeCCD\nPDObXFU4A0QFivwgvrKicc85JWtg4Y10UCcPRfAxpnT8afV9KY+sX6oZo1cKcCWL\n+SDc3Ikyd3Vf3nPvDy/ZqyGx/h5UpFKwgG9usYEzz57LVBxEnaOfRLN2E6sNvuse\nrr1yejFQXRqxBPj0PniiNyRy+dsFeCIflAa/cmzYch/lWgpM8pEj4XsYSu7lRMqx\nihmpmqpClZYx0Mu4CTaHW85S0aV6EhMOZgJmRnm+R+QNB4Q44ssnfyeb9TipKCE0\nkMCix1EqPoucgIekQzedwe32wDR0oGYMOdA1w44maDC9QaIKLjhu2f+8m+CqeD4Q\nvr97InCotkF5yL/eQtiSkUhUjg==\n-----END CERTIFICATE REQUEST-----\n"
