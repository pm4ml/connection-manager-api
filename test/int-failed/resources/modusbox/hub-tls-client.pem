-----BEGIN CERTIFICATE-----
MIIGgTCCBGmgAwIBAgIUERbavNDO8LfeDTS0jRaQ7Q6Fj6cwDQYJKoZIhvcNAQEL
BQAwVTELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAldBMRAwDgYDVQQHEwdTZWF0dGxl
MREwDwYDVQQKEwhNb2R1c2JveDEUMBIGA1UECxMLRW5naW5lZXJpbmcwHhcNMTkw
NjA1MjM1NTAwWhcNMjkwNjAyMjM1NTAwWjCBsjELMAkGA1UEBhMCVVMxCzAJBgNV
BAgTAldBMRAwDgYDVQQHEwdTZWF0dGxlMREwDwYDVQQKEwhNb2R1c2JveDEUMBIG
A1UECxMLRW5naW5lZXJpbmcxHTAbBgNVBAMTFGh1Yi5kZXYubW9kdXNib3guY29t
MTwwOgYJKoZIhvcNAQkBDC1jb25uZWN0aW9uLW1hbmFnZXItYWRtaW5AaHViLmRl
di5tb2R1c2JveC5jb20wggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQD1
oaAaAmS+/Yl9bO9p6Ljn5V/tWJxqOq8SOo78qNZDBktRdECreyEibaic3rt+0NaP
m6jyTYKxHyX9s094wvxMfJ+rk4etXWL8FfxXj8PLzZxo5SdSirRG+Xt1o1LPsxys
D6G8yNYPuv0U3yo7ijucbEuZnoAyVOFXBNyvxQZH1nm0oLhGOEeWMfk1NJZyGYtm
P6RCiGcrUbP3NPdlGEDKcSS/uMfaiLhgIk0y7/IqAJpRIfXUlT54clNd482oqyUw
3IA9HvVoCQbKolo48eiLzfeb0SQNNI+D50lPPNb/Lz9Puj6S+5wUugX5ulDUsDwS
BjhnmMDEH0fLs2ez7P5LDvhLj5p3kqtdyrB2kd766GFFEEELVsXhQctDn6Sm8Jjy
Kz4o1QJySLw0VvSuHQsE0re9tT9rmJSUoIOZ5u2YHbRMR+suoZ1LZv4K808PMcNM
G3qbrUWIDwzVaha/OATxyNjIQbb+VkhAcnqfMAnSQVxqXya7s6JuRAwSfEdCazEy
Lj81DiL5JVJapVv6fehgxIu987F7yRdimc66VtsbzuhwUvpj/px5Fgej28tXFJWa
5LpQ5NswXYkAVZFuSohEA6QR6KnG+QfLoYvP1SFUyrErg6bFDVssj8amTdQs+EHH
WVe3JEV4F+TViMZmUWmxP1qvekZ09xEJrR94WRTM4wIDAQABo4HqMIHnMA4GA1Ud
DwEB/wQEAwIFoDATBgNVHSUEDDAKBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB0G
A1UdDgQWBBRu+mqDejGqPwBoW3t+B3Mb560VPDAfBgNVHSMEGDAWgBQJP5EtJFht
yMYkyJ+NSnYb3qsl+DByBgNVHREEazBpghVodWIxLmRldi5tb2R1c2JveC5jb22C
FWh1YjIuZGV2Lm1vZHVzYm94LmNvbYEtY29ubmVjdGlvbi1tYW5hZ2VyLWFkbWlu
QGh1Yi5kZXYubW9kdXNib3guY29thwSjCgUYhwSjCgUVMA0GCSqGSIb3DQEBCwUA
A4ICAQBjOEMGovGVIWdjRnzTL9qCFZheni48DKe95S9eNQwp5BEtLzyHxkdnEsZb
M3e55kdknRc4HwtoNabiazhnk6hCBqQNAeI3JxagdMf7HPWnPb2QtTgX9hltPgDz
zWsbAlCJUrlZeMNmyT8yOAtCvdUVUMMTepiEMJwvavNIT5bwdXvd19HZU07YOg5s
btj+PGW+sOuGodPhd29QRe3oP8GXwkxSVSM+YWgUsbQq3rOxVhMEWVHnBejX01D2
/eAk1mVmgF63GgvyboYymdnc/1mNJZNJUOBnk9vSENmDVCIRGrql1g2tQHHtfwnj
rFcDDmc+Nek07zUz+C7K38XiqXOS33BrVxh8caiNC8aSfi6LLrhvWgXbsKBc/UVI
/OzfRw9i70uOAH7Ys+RBxOiy2QKx4P9oOygARlxUpz5tHVcSw14njMfBOghqSLO5
Zcbzr3JF21AgswYSJ2w4PYE9F3poCpdXfffD7bjEbKJVY0cIAqGMyL8pJY6YUt75
fXBbasHXSXAH4mxTHQg3U7Fj+k3qlPHu0yHRXmKbdbFdlP4ywgVrhzS+EunmeXWK
vB6A8l+1ADXq5HydhRwdTkfZm7uPql4xAHqyXRPEVu9tSElutMS7vo2a97qaTK6X
zaoGbLdioYWImIWsl86zeBQ67TwKYt5fHke9mCx53G6/RVmEzQ==
-----END CERTIFICATE-----
