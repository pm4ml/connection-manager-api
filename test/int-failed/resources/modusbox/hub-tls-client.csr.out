Certificate Request:
    Data:
        Version: 0 (0x0)
        Subject: C=US, ST=WA, L=Seattle, O=Modusbox, OU=Engineering, CN=hub.dev.modusbox.com/emailAddress=connection-manager-admin@hub.dev.modusbox.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:f5:a1:a0:1a:02:64:be:fd:89:7d:6c:ef:69:e8:
                    b8:e7:e5:5f:ed:58:9c:6a:3a:af:12:3a:8e:fc:a8:
                    d6:43:06:4b:51:74:40:ab:7b:21:22:6d:a8:9c:de:
                    bb:7e:d0:d6:8f:9b:a8:f2:4d:82:b1:1f:25:fd:b3:
                    4f:78:c2:fc:4c:7c:9f:ab:93:87:ad:5d:62:fc:15:
                    fc:57:8f:c3:cb:cd:9c:68:e5:27:52:8a:b4:46:f9:
                    7b:75:a3:52:cf:b3:1c:ac:0f:a1:bc:c8:d6:0f:ba:
                    fd:14:df:2a:3b:8a:3b:9c:6c:4b:99:9e:80:32:54:
                    e1:57:04:dc:af:c5:06:47:d6:79:b4:a0:b8:46:38:
                    47:96:31:f9:35:34:96:72:19:8b:66:3f:a4:42:88:
                    67:2b:51:b3:f7:34:f7:65:18:40:ca:71:24:bf:b8:
                    c7:da:88:b8:60:22:4d:32:ef:f2:2a:00:9a:51:21:
                    f5:d4:95:3e:78:72:53:5d:e3:cd:a8:ab:25:30:dc:
                    80:3d:1e:f5:68:09:06:ca:a2:5a:38:f1:e8:8b:cd:
                    f7:9b:d1:24:0d:34:8f:83:e7:49:4f:3c:d6:ff:2f:
                    3f:4f:ba:3e:92:fb:9c:14:ba:05:f9:ba:50:d4:b0:
                    3c:12:06:38:67:98:c0:c4:1f:47:cb:b3:67:b3:ec:
                    fe:4b:0e:f8:4b:8f:9a:77:92:ab:5d:ca:b0:76:91:
                    de:fa:e8:61:45:10:41:0b:56:c5:e1:41:cb:43:9f:
                    a4:a6:f0:98:f2:2b:3e:28:d5:02:72:48:bc:34:56:
                    f4:ae:1d:0b:04:d2:b7:bd:b5:3f:6b:98:94:94:a0:
                    83:99:e6:ed:98:1d:b4:4c:47:eb:2e:a1:9d:4b:66:
                    fe:0a:f3:4f:0f:31:c3:4c:1b:7a:9b:ad:45:88:0f:
                    0c:d5:6a:16:bf:38:04:f1:c8:d8:c8:41:b6:fe:56:
                    48:40:72:7a:9f:30:09:d2:41:5c:6a:5f:26:bb:b3:
                    a2:6e:44:0c:12:7c:47:42:6b:31:32:2e:3f:35:0e:
                    22:f9:25:52:5a:a5:5b:fa:7d:e8:60:c4:8b:bd:f3:
                    b1:7b:c9:17:62:99:ce:ba:56:db:1b:ce:e8:70:52:
                    fa:63:fe:9c:79:16:07:a3:db:cb:57:14:95:9a:e4:
                    ba:50:e4:db:30:5d:89:00:55:91:6e:4a:88:44:03:
                    a4:11:e8:a9:c6:f9:07:cb:a1:8b:cf:d5:21:54:ca:
                    b1:2b:83:a6:c5:0d:5b:2c:8f:c6:a6:4d:d4:2c:f8:
                    41:c7:59:57:b7:24:45:78:17:e4:d5:88:c6:66:51:
                    69:b1:3f:5a:af:7a:46:74:f7:11:09:ad:1f:78:59:
                    14:cc:e3
                Exponent: 65537 (0x10001)
        Attributes:
        Requested Extensions:
            X509v3 Subject Alternative Name: 
                DNS:hub1.dev.modusbox.com, DNS:hub2.dev.modusbox.com, email:connection-manager-admin@hub.dev.modusbox.com, IP Address:163.10.5.24, IP Address:163.10.5.21
    Signature Algorithm: sha512WithRSAEncryption
         f3:18:18:98:4e:3c:20:5e:7b:b5:fb:2e:f2:f0:74:4a:2e:7c:
         b4:11:83:29:54:ea:c0:7f:fb:12:ad:52:ac:87:61:31:4b:7a:
         35:d1:5c:d5:f8:ac:9e:c0:fb:12:0d:f7:b0:fd:2e:d5:0f:15:
         aa:65:c2:65:66:b1:09:5d:6f:d9:81:91:a4:0e:3b:15:4e:00:
         92:31:0f:c8:54:f0:d7:2b:24:08:13:30:96:07:c3:5e:e5:49:
         c2:f6:82:23:52:50:79:97:74:af:95:de:10:d0:3b:52:ea:11:
         3a:dd:ac:f3:f1:32:18:4a:2b:f0:20:4f:0e:1b:f4:d7:43:1a:
         ad:8e:32:4c:0b:6b:02:70:70:3d:29:9b:19:67:07:ac:d6:f7:
         b3:5d:1d:cb:23:2e:11:f0:60:4b:62:95:55:0c:68:90:9c:db:
         17:9c:8a:9c:c7:69:c7:de:43:ff:97:4d:0f:2b:d2:a4:fe:33:
         89:c4:a1:9b:41:bb:3b:f6:7b:5a:00:9f:40:5e:e4:07:63:18:
         35:de:ae:29:a4:61:be:48:25:a4:13:1d:cb:0d:04:cf:8b:e5:
         96:86:57:04:4e:7a:6f:91:3d:63:00:7d:52:59:5d:e5:77:c7:
         3b:4d:c5:cf:a5:c8:f4:0d:84:da:05:8a:2f:2f:a6:da:78:20:
         83:3c:33:9b:5c:55:38:03:44:05:8a:fc:20:be:b2:a2:71:cf:
         39:25:6b:60:e1:8d:74:50:27:0f:45:f0:31:a6:74:fc:69:f5:
         7d:29:8f:ac:5f:aa:19:a3:57:0a:70:25:8b:f9:20:dc:dc:89:
         32:77:75:5f:de:73:ef:0f:2f:d9:ab:21:b1:fe:1e:54:a4:52:
         b0:80:6f:6e:b1:81:33:cf:9e:cb:54:1c:44:9d:a3:9f:44:b3:
         76:13:ab:0d:be:eb:1e:ae:bd:72:7a:31:50:5d:1a:b1:04:f8:
         f4:3e:78:a2:37:24:72:f9:db:05:78:22:1f:94:06:bf:72:6c:
         d8:72:1f:e5:5a:0a:4c:f2:91:23:e1:7b:18:4a:ee:e5:44:ca:
         b1:8a:19:a9:9a:aa:42:95:96:31:d0:cb:b8:09:36:87:5b:ce:
         52:d1:a5:7a:12:13:0e:66:02:66:46:79:be:47:e4:0d:07:84:
         38:e2:cb:27:7f:27:9b:f5:38:a9:28:21:34:90:c0:a2:c7:51:
         2a:3e:8b:9c:80:87:a4:43:37:9d:c1:ed:f6:c0:34:74:a0:66:
         0c:39:d0:35:c3:8e:26:68:30:bd:41:a2:0a:2e:38:6e:d9:ff:
         bc:9b:e0:aa:78:3e:10:be:bf:7b:22:70:a8:b6:41:79:c8:bf:
         de:42:d8:92:91:48:54:8e
